app/assets/stylesheets/ama_layout/layout_components/siteheader.scss

```diff
@@ -13,6 +13,12 @@
   top: 0;
   width: 100%;
   z-index: 9999;
+
+  &--absolute-mobile{
```

Comment:
@kaytcampbell The header will need to be absolute of mobile only for when there is an app banner, since when it is fixed it wont push down the elements on android. I think it still needs to be fixed on mobile for other apps because then it will scroll with the screen, but on RR most of the pages dont scroll so we're not missing much.